api/v1/docker/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+# This Makefile is expected to run inside cilium/hubble-proto image to generate
+# the proto files.
+
+PROTOC ?= protoc
+
+HUBBLE_PROTO_SOURCES := \
+	./flow/flow.proto \
+	./peer/peer.proto \
+	./observer/observer.proto \
+	./relay/relay.proto
+
+HUBBLE_GO_TARGETS := $(HUBBLE_PROTO_SOURCES:.proto=.pb.go) $(HUBBLE_PROTO_SOURCES:.proto=.pb.json.go)
+
+HUBBLE_PROTO_PATH := .
+
+HUBBLE_PROTOC_PLUGINS := --plugin=/proto/protoc-gen-go
+HUBBLE_PROTOC_PLUGINS += --plugin=/proto/protoc-gen-go-grpc
+HUBBLE_PROTOC_PLUGINS += --plugin=/proto/protoc-gen-go-json
```

Comment:
So `protoc-gen-validate` is gone now? There's no mention of this in the commit message.

Reply:
FWIW: We never used it. I think we might as well drop it.

Reply:
I'm OK with dropping it, especially if we don't use it but I don't like that it's being silently removed.

Reply:
I will mention the change in the commit. Yes, it's because it wasn't used.